core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Manager.java

```diff
@@ -173,14 +174,17 @@ public AuthSession tableSession(RESTClient sharedClient, Map<String, String> pro
       sessionCache = newSessionCache(name, properties);
     }
 
+    String oauth2ServerUri = properties.get(OAuth2Properties.OAUTH2_SERVER_URI);
```

Comment:
this should be falling back to `ResourcePaths.tokens()` because this property is not mandatory